common/src/IO/EntParser.cpp

```diff
@@ -622,6 +622,36 @@ std::optional<Color> EntParser::parseColor(
   return Color::parse(parseString(element, attributeName, status));
 }
 
+std::optional<bool> EntParser::parseBoolean(
+  const tinyxml2::XMLElement& element,
+  const std::string& attributeName,
+  ParserStatus& /* status */)
+{
+  std::string strValue = element.Attribute(attributeName.c_str());
+  if (strValue.empty())
+  {
+    return std::nullopt;
+  }
+
+  if (strValue == "True")
+  {
+    return true;
+  }
+  else if (strValue == "False")
```

Comment:
You don't need the `else` here due to the unconditional return from the previous if block.

Reply:
fun fact, that's the kind of things that usually makes me complain about dumb code, when I see a `else` after a `if(){return;}` :D